content/en/docs/next/install.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Install
+weight: 1150
+description: Instructions for downloading and building different versions of etcd
+---
+
+## System requirements
+
+The etcd performance benchmarks run etcd on 8 vCPU, 16GB RAM, 50GB SSD GCE
+instances, but any relatively modern machine with low latency storage and a few
+gigabytes of memory should suffice for most use cases. Applications with large
+v2 data stores will require more memory than a large v3 data store since data is
+kept in anonymous memory instead of memory mapped from a file. For running etcd
+on a cloud provider, see the [Example hardware configuration][example-hardware-configurations]
+documentation.
+
+## Download the pre-built binary
+
+The easiest way to get etcd is to use one of the pre-built release binaries
+which are available for OSX, Linux, Windows, appc, and Docker. Instructions for
+using these binaries are on the [GitHub releases page][github-release].
+
+## Build the latest version
+
+For those wanting to try the very latest version, build etcd from the `main`
+branch. [Go](https://golang.org/) version 1.13+ is required to build the latest
+version of etcd. To ensure etcd is built against well-tested libraries, etcd
+vendors its dependencies for official release binaries. However, etcd's
+vendoring is also optional to avoid potential import conflicts when embedding
+the etcd server or using the etcd client.
+
+To build `etcd` from the `main` branch without a `GOPATH` using the official
+`build` script:
+
+```sh
+$ git clone https://github.com/etcd-io/etcd.git
+$ cd etcd
+$ ./build
+```
+
+To build a modularized `etcd` from the `main` branch via `go get`:
+
+{{% alert title="Note" color="info" %}}
+<!-- TODO: find out how to set the GOPATH -->
+GOPATH should be set. If it isn't, [learn how here](https://github.com/golang/go/wiki/SettingGOPATH).
+{{% /alert %}}
+
+```sh
+$ go get -v go.etcd.io/etcd/v3
+$ go get -v go.etcd.io/etcd/v3/etcdctl
+```
```

Comment:
@spzala, @ptabor, are these still the correct instructions for the most recent version of go?

I've setup go with a GOPATH, and when I do the `go get` only the `~/go/pkg` directory is created.

Reply:
this is currently what is on the site:

```
# GOPATH should be set
$ echo $GOPATH
/Users/example/go
$ go get -v go.etcd.io/etcd
$ go get -v go.etcd.io/etcd/etcdctl
```

But this doesn't generate the `bin` folder either.

Reply:
some env info:
```
➜  ~ go version
go version go1.16.4 linux/amd64
➜  ~ echo $GOPATH
/home/natew-cncf/go
➜  ~ ls ~/go
pkg
```

Reply:
@ptabor Should we add `etcdutil` as well? (newly introduced in 3.5)